SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -142,7 +144,7 @@ class UITable extends Component {
         rowData
       );
 
-      return <td key={tdIndex} className={col.props.className}>{cell}</td>;
+      return <td key={col.props.id} className={col.props.className}>{cell}</td>;
```

Comment:
not sure we necessarily want to combine them -- the whole point of `key` is to help React optimize the rendering (i.e. reordered or destroyed instead of all of them being clobbered). I'm OK with the original approach as long as we have some sort of note somewhere that `props.id` should be unique